src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -1319,11 +1319,9 @@ class Simpletask : ThemedActivity() {
             }
             cb.isChecked = completed
 
-            val relAge = getRelativeAge(task, TodoApplication.app)
-            val relDue = getRelativeDueDate(task, m_app, ContextCompat.getColor(m_app, android.R.color.holo_green_light),
-                    ContextCompat.getColor(m_app, android.R.color.holo_red_light),
-                    Config.hasColorDueDates())
-            val relativeThresholdDate = getRelativeThresholdDate(task, TodoApplication.app)
```

Comment:
Again, don't need all these parms. Also, passing the app sometimes as a TodoApplication and sometimes as a Context was weird. Let's always pass m_app.